pkgs/by-name/nv/nv-codec-headers/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  callPackage,
+  stdenvNoCC,
+  # Configurable options
+  majorVersion ? "9",
```

Comment:
So just to confirm, `nv-codec-headers == nv-codec-headers-9` right?

Reply:
Yes.
This is the default since, well, a long time.

On the other hand, the only package that uses the default value for nvch is Handbrake:

https://github.com/NixOS/nixpkgs/blob/1d9c2c9b3e71b9ee663d11c5d298727dace8d374/pkgs/applications/video/handbrake/default.nix#L272